### ISSUE TITLE
[Rogue] Sap => Stuck mob

### ISSUE THREAD

**Issue by nullfoxh** [ccshiro/cc-buglist#671]
This bug was reported by cdreipo as a comment on my other distract bugreport, but I felt it needed a separate report.

If you first distract a mob and then sap it the mob gets stuck and will not move from that position. It remains killable however.

See choppy video (20mb file, sorry):
http://webm.host/5e5db/

Update: A lot more details in comments below!

**Comment by Chulain** [comment]
https://github.com/ccshiro/cc-buglist/issues/647 ?

**Comment by nullfoxh** [comment]
Yes Chulain, that seems to be the same bug. Did not come across it.
My video however contains the info missing from the other report. :)

**Comment by petaal** [comment]
I got some screenshots of .movegens --verbose. Distracted the mob, sapped and attacked with Throw. Got two different outputs with different mobs:
![wowscrnshot_060715_142134](https://cloud.githubusercontent.com/assets/11743711/8023740/e2ef7a1c-0d20-11e5-866f-fa9705a114ff.jpg)
![wowscrnshot_060715_141713](https://cloud.githubusercontent.com/assets/11743711/8023742/ef884c90-0d20-11e5-94ca-e04a48d19217.jpg)

**Comment by nullfoxh** [comment]
I have been testing this issue with several mobs around outland. I found that distract is not needed to reproduce this issue, sap alone is enough.

Good news is I think I have found the key to fixing this issue! With this issue the mobs will randomly move about, like they would if they were not in combat. Only difference is, they will cast spells and face you. So it would seem sapping them stops their movement logic from switching from "idle" to "combat mode"!

The movegens/movement priority output also supports my theory as it is different for a mob that is idle, and one that has been engaged in combat (as common sense would dictate.) However, if you sap the mob first, its movement priority seems to remain in "idle mode" rather than switch to the "combat mode."

And lastly, here's some screenshots to back up my claims:
Movement prio for a mob as idle, who then was sapped and hit with ranged (it remained the same):
![wowscrnshot_060715_152225](https://cloud.githubusercontent.com/assets/12772977/8023993/cae7d148-0d2b-11e5-8f58-e02fac6e45b2.jpg)
And this is what the movement prio looks like for a mob that has not been sapped:
![wowscrnshot_060715_152110](https://cloud.githubusercontent.com/assets/12772977/8023994/dee85c9e-0d2b-11e5-9304-17cdd13c40fa.jpg)

Same results here, sapped first and prio remains unchanged:
![wowscrnshot_060715_151806](https://cloud.githubusercontent.com/assets/12772977/8024014/88128916-0d2c-11e5-8d30-3bc99a742104.jpg)
And a mob that was not sapped:
![wowscrnshot_060715_151729](https://cloud.githubusercontent.com/assets/12772977/8024017/97b7ccd2-0d2c-11e5-82c5-d629140c3853.jpg)
